JASP-Engine/JASP/R/ttestbayesianonesample.R

```diff
@@ -4517,9 +4517,8 @@ TTestBayesianOneSample <- function(dataset=NULL, options, perform="run", callbac
                             oneSided = FALSE, options) {
 
   tValue <- unname(t.test(x, y, paired = paired, var.equal = TRUE)$statistic)
-
-  n1 <- length(x)
-  n2 <- ifelse(paired, 0, length(y))
+  n1 <- as.numeric(length(x))
```

Comment:
Let's add a comment to this

# length(x) is an integer and integer times integer leads can lead to an overflow
# numeric multiplication is more robust in R